library/core/src/cell.rs

```diff
@@ -1570,13 +1570,14 @@ impl<T: ?Sized + fmt::Display> fmt::Display for RefMut<'_, T> {
 /// 2. A `&mut T` reference may be released to safe code provided neither other `&mut T` nor `&T`
 /// co-exist with it. A `&mut T` must always be unique.
 ///
-/// Note that while mutating or mutably aliasing the contents of an `&UnsafeCell<T>` is
+/// Note that whilst mutating the contents of an `&UnsafeCell<T>` (even while other
+/// `&UnsafeCell<T>` references alias the cell) is
 /// ok (provided you enforce the invariants some other way), it is still undefined behavior
 /// to have multiple `&mut UnsafeCell<T>` aliases. That is, `UnsafeCell` is a wrapper
-/// designed to have a special interaction (_w.r.t._ the compiler) with _shared_
-/// accesses (_i.e._, through an `&UnsafeCell<_>` reference): there is no magic whatsoever
-/// when dealing with _exclusive_ access (an `&mut UnsafeCell<_>`): neither the cell
-/// nor the wrapped value may be aliased for the duration of that `&mut` borrow.
+/// designed to have a special interaction with _shared_ accesses (_i.e._, through an
+/// `&UnsafeCell<_>` reference): there is no magic whatsoever when dealing with _exclusive_
+/// accesses (_e.g._, through an `&mut UnsafeCell<_>`): neither the cell nor the wrapped value
+/// may be aliased for the duration of that `&mut` borrow.
 ///
 ///   - This is showcased by the [`.get_mut()`] accessor, which is a non-`unsafe`
```

Comment:
This is still an odd bullet point. A singleton enumeration is hardly worth being called an enumeration. ;)

I think this can just be a sentence at the end of the previous paragraph.